src/Microsoft.Health.Fhir.Core/Features/Search/SearchService.cs

```diff
@@ -139,7 +139,7 @@ protected SearchService(ISearchOptionsFactory searchOptionsFactory, IFhirDataSto
 
             if (at != null)
             {
-                queryParameters.Add(Tuple.Create(SearchParameterNames.LastUpdated, at.ToString()));
+                queryParameters.Add(Tuple.Create(KnownQueryParameterNames.At, at.ToString()));
```

Comment:
Since at will not be supported, should it still be mutually exclusive with since & before?

Reply:
Good question. As John has mentioned in one of my older PR, that we should support _At and _since/_before together. So I have created UserStory #[76521](https://microsofthealth.visualstudio.com/Health/_boards/board/t/Resolute/Stories/?workitem=76521) for this. And there is an open [PR ](https://github.com/microsoft/fhir-server/pull/1183)to enable _at parameter for SqlServer. So considering these, I have not changed _at and _since/_before relation as part of this PR.